lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -15,6 +15,8 @@ class KubernetesResource
     GLOBAL = false
     TIMEOUT = 5.minutes
     LOG_LINE_COUNT = 250
+    SERVER_DRY_RUN_DISABLED_ERROR =
+      /(unknown flag: --server-dry-run)|(doesn't support dry-run)|(dryRun alpha feature is disabled)/
```

Comment:
Does it look OK to you?

Reply:
I'll make a 🎩against different version, better to cover from v1.11 to v1.15. Will list the result

Reply:
The updated 🎩 is https://github.com/Shopify/kubernetes-deploy/pull/553#issuecomment-531903409